app/src/sharedTest/java/org/oppia/app/profileprogress/ProfileProgressFragmentTest.kt

```diff
@@ -266,11 +275,9 @@ class ProfileProgressFragmentTest {
       )
     }
   }
-  /* ktlint-enable max-line-length */
 
-  /* ktlint-disable max-line-length */
   @Test
-  fun testProfileProgressFragmentNoProgress_recyclerViewItem0_checkOngoingTopicsString_descriptionIsCorrect() {
+  fun testProfileProgressFragmentNoProgress_recyclerViewItem0_checkOngoingTopicsString_descriptionIsCorrect() {  // ktlint-disable max-line-length
```

Comment:
`Unnecessary space(s)` there are 2 spaces before `// ktlint-disable max-line-length`. it should be one else ktlint will fail

Reply:
Done.

Reply:
seems line not resolved.

Reply:
I think it got reverted. Now its done.